mteb/models/overview.py

```diff
@@ -21,6 +21,8 @@
     mxbai_models,
     nomic_models,
     openai_models,
+    promptriever_models,
+    repllama_models,
```

Comment:
I was using these as a test and noticed they weren't here. Adding them but lmk if they were supposed to be elsewhere.

Reply:
This is perfect